datahub-graphql-core/src/main/resources/gms.graphql

```diff
@@ -1435,6 +1456,11 @@ type Chart implements Entity {
     The structured tags associated with the chart
     """
     globalTags: GlobalTags
+
+    """
```

Comment:
What is downstream of a chart? Another chart? 

I'd think that Dashboard would need to have downstream lineage to charts

Reply:
You're correct, dashboards are downstream of charts. Because of this, chart's downstream properties are dashboards. Dashboards have no downstreams, since nothing is dependent on a dashboard in our model.